templates/Import/index.html.twig

```diff
@@ -15,15 +15,20 @@
             {% include 'Import/_information.html.twig' %}
 
             {{ 'import.page_description'|trans }}
-            <ul>
+            <div class="row">
                 {% for import in imports %}
-                    <li>
-                        <h5>{{ import.name }}</h5>
-                        <blockquote>{{ import.description|trans|raw }}</blockquote>
```

Comment:
No more description?

Reply:
+1 I was just asking it myself

Reply:
<img width="1324" alt="Capture d’écran 2024-11-14 à 16 15 17" src="https://github.com/user-attachments/assets/1d729045-0860-46a6-93f2-dd5dfe70327c">

I removed it to avoid this. 

And the description is available on each Import page.